doc/ENCODINGS

```diff
@@ -61,14 +61,15 @@ If you use rules without --internal-codepage, some wordlist rules may cut
 UTF-8 multibyte sequences in the middle, resulting in garbage. You can reject
 such rules with -U to have them in use only when an internal codepage is used.
 
-Caveats:
-Please note that for convenience, any rule, mask or placeholder given in UTF-8
+Please note that for convenience any rule, mask or placeholder given in UTF-8
 on command-line or in a config file will be silently translated to whatever
-internal codepage is in use.  This will normally result in just what you'd
-expect but it could sometimes lead to confusion.  For example, if you run that
-"scç" rule with the internal codepage set to CP866, the "ç" can't be translated
-and will currently lead to some other undefined substitution with no warning.
+internal codepage is in use at the time.  For this to happen with rules, you
+must prepend them with "-U".  If such conversion fails for a rule, that rule
+will be rejected (silently, but logged).  Thus you can have rules for various
+codepages such as "-U sRЯ" and "-U sc©" in place - and only one of them, both,
+or none will be used, depending on what internal codepage is active.
```

Comment:
Since the functionality is now conditional rather than always enabled (right?), perhaps the "will" in "will be silently translated" should be changed to "can". Also, maybe replace "translated" with "converted" for consistency with the word "conversion" in what follows. Then you describe what it takes to enable the functionality for rules. You can add at the end that this conversion always happens for masks and placeholders (does it?)

Reply:
I will revisit this - don't merge yet.

Reply:
Rewritten, and separated rules from masks better